src/app.js

```diff
@@ -122,7 +137,7 @@ class SAFEApp extends EventEmitter {
     });
 
     if (!hasCorrectProperties) {
-      throw makeFfiError(errConst.MALFORMED_APP_INFO.code, errConst.MALFORMED_APP_INFO.msg);
+      throw new Error(errConst.MALFORMED_APP_INFO.msg);
```

Comment:
So we are not returning error codes from safe-app-nodejs layer but just error messages? if so, do we need the `code` attribute in the `errConst` objects?
If we are using error codes, we would preferably add them to the documentation.

Reply:
I don't find the error codes useful but I was continuing with what was already established for consistency. The case where error codes are useful is when we are expecting certain native errors in `webFetch` and can just check for the presence of code `-103`, for example.
I suppose a similar case may be useful to developers building on top of web API or safe_app_nodejs later on.

Reply:
If we are not adding error codes now, then we sholdn't need the error code in the `src/error_const.js` objects, right?